packages/renderer/src/lib/container/ContainerColumnName.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+import { router } from 'tinro';
+
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+import { ContainerGroupInfoTypeUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+
+function displayContainersCount(containerGroup: ContainerGroupInfoUI) {
+  let result = containerGroup.allContainersCount + ' container' + (containerGroup.allContainersCount > 1 ? 's' : '');
+  if (containerGroup.containers.length !== containerGroup.allContainersCount) {
+    result += ` (${containerGroup.allContainersCount - containerGroup.containers.length} filtered)`;
+  }
+  return result;
+}
+
+function openGroupDetails(containerGroup: ContainerGroupInfoUI) {
+  if (!containerGroup.engineId) {
+    return;
+  }
+  if (containerGroup.type === ContainerGroupInfoTypeUI.POD) {
+    router.goto(`/pods/podman/${encodeURI(containerGroup.name)}/${encodeURIComponent(containerGroup.engineId)}/logs`);
+  } else if (containerGroup.type === ContainerGroupInfoTypeUI.COMPOSE) {
+    router.goto(`/compose/details/${encodeURI(containerGroup.name)}/${encodeURI(containerGroup.engineId)}/logs`);
+  }
+}
+
+function openContainerDetails(container: ContainerInfoUI) {
+  router.goto(`/containers/${container.id}/`);
+}
+</script>
+
+{#if 'type' in object && (object.type === ContainerGroupInfoTypeUI.POD || object.type === ContainerGroupInfoTypeUI.COMPOSE)}
```

Comment:
`'type' in object && (object.type === ContainerGroupInfoTypeUI.POD || object.type === ContainerGroupInfoTypeUI.COMPOSE)` could be replaced by an utility method isContainerGroup() or something similar

Reply:
I forgot that I simplified and avoided this ever being called with a standalone container parent, so I was able to remove `object.type ===` in the new commit. But yes, a utility method might be useful in future.